flixel/FlxSprite.hx

```diff
@@ -57,11 +57,17 @@ class FlxSprite extends FlxObject
 	 */
 	public var useFramePixels(default, set):Bool = true;
 	
-	// TODO: rename this to smoothing...
 	/**
 	 * Controls whether the object is smoothed when rotated, affects performance.
 	 */
-	public var antialiasing(default, set):Bool = false;
+	public var smoothing(default, set):Bool = false;
+	
+	/**
+	 * Controls whether the object is smoothed when rotated, affects performance.
+	 */
+	@:deprecated
```

Comment:
We should probably put a deprecation message here ("Use smoothing instead" or something like that.)